nsv13/code/modules/clothing/helmet_camera.dm

```diff
@@ -64,3 +60,26 @@
 		updating = TRUE
 		addtimer(CALLBACK(src, PROC_REF(do_camera_update), oldLoc), SILICON_CAMERA_BUFFER)
 #undef SILICON_CAMERA_BUFFER
+
+/obj/item/clothing/head/helmet/proc/install_camera(obj/item/I, mob/user, faction)
+	if(builtInCamera) //You already have a camera
+		return FALSE
+	if(!I || !user) //The camera isn't being placed by a person
+		builtInCamera = new /obj/machinery/camera(src)
+		builtInCamera.c_tag = "Helmet Cam #[rand(0,999)]"
+		if(faction == "Syndicate")
+			builtInCamera.network = list("syndicate")
+		else
+			builtInCamera.network = list("ss13")
+			builtInCamera.internal_light = FALSE
```

Comment:
`internal_light = FALSE` is likely indented one too far since it should do it in either case.

Reply:
Yep...